packages/web-components/src/tabs/tabs.styles.ts

```diff
@@ -0,0 +1,258 @@
+import { css } from '@microsoft/fast-element';
+import {
+  borderRadiusCircular,
+  borderRadiusMedium,
+  colorNeutralForeground1,
+  colorNeutralForeground1Hover,
+  colorNeutralForeground2,
+  colorNeutralForegroundDisabled,
+  colorSubtleBackgroundHover,
+  colorSubtleBackgroundPressed,
+  curveDecelerateMax,
+  durationSlow,
+  fontFamilyBase,
+  fontSizeBase300,
+  fontSizeBase400,
+  fontWeightRegular,
+  lineHeightBase300,
+  lineHeightBase400,
+  spacingHorizontalMNudge,
+  spacingHorizontalSNudge,
+  spacingVerticalL,
+  spacingVerticalMNudge,
+  spacingVerticalS,
+  spacingVerticalSNudge,
+  spacingVerticalXXS,
+  strokeWidthThicker,
+} from '../theme/design-tokens.js';
+import { TabTokenNames } from './tabs.js';
+
+export const styles = css`
+  :host {
+    display: grid;
+    box-sizing: border-box;
+    font-family: ${fontFamilyBase};
+    font-size: ${fontSizeBase300};
+    line-height: ${lineHeightBase300};
+    color: ${colorNeutralForeground2};
+    grid-template-columns: auto 1fr auto;
+    grid-template-rows: auto 1fr;
+  }
+
+  :host([disabled]) {
+    cursor: not-allowed;
+    color: ${colorNeutralForegroundDisabled} !important;
```

Comment:
Are we running into specificity issues with this selector? Problem with important is its going to lock out any overrides.

Reply:
There is no specificity issue here. The reason important is added is so that this color can't be overridden, disabled always takes priority. (I though this was the way disabled was supposed to work)

Reply:
Got it, its just not something we have we have done in other libraries but I guess if we feel this shouldn't ever be configurable from the consumer I'm fine with it. @chrisdholt for visibility

Reply:
Totally fine with whatever is the pattern for our WC library. In the past i have used !important to prevent users from overriding in certain cases, this seemed like appropriate use to me.

Reply:
I went ahead and removed the bang importants, since it seems like it isn't a pattern we're using.